x-pack/plugins/osquery/public/shared_components/osquery_action/use_is_osquery_available.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
do we need that as we do have [use_is_osquery_available_simple.tsx](https://github.com/elastic/kibana/pull/123576/files#diff-d471fac3176ef2feba7a7ffab80df558562886b4681b2a512e69ad3dd6d2f925)?

Reply:
This hook exposes much more thanks to react-query. I didn't want to combine these 2 hooks together. What do you think?